### ISSUE TITLE
Training stability & progress

### ISSUE THREAD

**Issue by Cpruce** [alexlee-gk/video_prediction#31]
@alexlee-gk 

Thanks for an awesome repo and architecture! I am currently training on my own dataset with batch size 8 and have been playing around with lower learning rates.

While the predicted frames are decent, my metrics seem to have a high variance, indicating the training process can be made better for this data. 

Do you have any suggestions on how to improve training stability and/or results, besides further decreasing the learning rate? Here are some screenshots from the tensorboard graphs.

<img width="517" alt="Screen Shot 2019-05-18 at 8 18 19 PM" src="https://user-images.githubusercontent.com/2712171/57977330-2645fc00-79ab-11e9-8552-33e7b3bf8850.png">
<img width="448" alt="Screen Shot 2019-05-18 at 8 18 45 PM" src="https://user-images.githubusercontent.com/2712171/57977331-2645fc00-79ab-11e9-8b06-de9517ce1c87.png">
<img width="457" alt="Screen Shot 2019-05-18 at 8 19 00 PM" src="https://user-images.githubusercontent.com/2712171/57977332-2645fc00-79ab-11e9-9349-6251d94a3ced.png">
<img width="436" alt="Screen Shot 2019-05-18 at 8 19 48 PM" src="https://user-images.githubusercontent.com/2712171/57977333-26de9280-79ab-11e9-93f1-672f76b25631.png">
<img width="464" alt="Screen Shot 2019-05-18 at 8 20 03 PM" src="https://user-images.githubusercontent.com/2712171/57977334-26de9280-79ab-11e9-9840-58f3fc3d42c0.png">

**Comment by mwy691222563** [comment]
> @alexlee-gk
> 
> Thanks for an awesome repo and architecture! I am currently training on my own dataset with batch size 8 and have been playing around with lower learning rates.
> 
> While the predicted frames are decent, my metrics seem to have a high variance, indicating the training process can be made better for this data.
> 
> Do you have any suggestions on how to improve training stability and/or results, besides further decreasing the learning rate? Here are some screenshots from the tensorboard graphs.
> 
> <img alt="Screen Shot 2019-05-18 at 8 18 19 PM" width="517" src="https://user-images.githubusercontent.com/2712171/57977330-2645fc00-79ab-11e9-8552-33e7b3bf8850.png">
> 
> <img alt="Screen Shot 2019-05-18 at 8 18 45 PM" width="448" src="https://user-images.githubusercontent.com/2712171/57977331-2645fc00-79ab-11e9-8b06-de9517ce1c87.png">
> 
> <img alt="Screen Shot 2019-05-18 at 8 19 00 PM" width="457" src="https://user-images.githubusercontent.com/2712171/57977332-2645fc00-79ab-11e9-9349-6251d94a3ced.png">
> 
> <img alt="Screen Shot 2019-05-18 at 8 19 48 PM" width="436" src="https://user-images.githubusercontent.com/2712171/57977333-26de9280-79ab-11e9-93f1-672f76b25631.png">
> 
> <img alt="Screen Shot 2019-05-18 at 8 20 03 PM" width="464" src="https://user-images.githubusercontent.com/2712171/57977334-26de9280-79ab-11e9-9840-58f3fc3d42c0.png">

Hello, how to run code on my own datasets, could you give me some suggestions,  I changed kth_dataset to generate my tfrecords, is this enough?  Do I need to rewrite some other funs?

thank you

**Comment by mwy691222563** [comment]
@Cpruce 

How to run code on my datasets? I changed  kth_dataset, do I need to rewrite some other funs? 

Thx

**Comment by eslambakr** [comment]
I have the same question, as while training on my own data-set I got the same curves.
I am wondering why the generator loss is increasing?!

Any updates regarding this issue?

Thanks in advance.
